miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -140,7 +140,8 @@ internal class MiniAppDownloader(
             else {
                 val apiResponse = apiClient.fetchMiniAppManifest(appId, versionId)
                 val latestManifest = prepareMiniAppManifest(apiResponse, versionId)
-                manifestApiCache.storeManifest(appId, versionId, latestManifest)
+                if (!apiClient.isPreviewMode)
```

Comment:
Do we need to check preview mode here?

Reply:
I think so, the manifest could be changed so it should not be stored. cc @corycaywood

Reply:
Yeah, my perspective is downloading a miniapp is prior to downloading miniapp's manifest. So, if we already check for miniapp, we don't need to check for the manifest. 
What do you think @rleojoseph

Reply:
It is about storing, not downloading.

Reply:
I forgot this on iOS, but it will soon be solved be the store manifest as file feature